requirements_alt.txt

```diff
@@ -0,0 +1,15 @@
+tqdm==4.36.1
+requests==2.22.0
+tensorpack==0.9.8
+setuptools==41.2.0
+six==1.12.0
+opencv_python==4.0.0.21
+tensorflow_gpu==1.12.0
+matplotlib==3.1.1
+psutil==5.6.3
+scipy==1.3.1
+ipython==7.8.0
+Pillow==6.2.1
+protobuf==3.10.0
+rospkg==1.1.10
+tensorflow==2.0.0
```

Comment:
@lordtt13 Just making sure but is it fully compatible with TF 2.0?

Reply:
I'll check by running the container in an env and if not i'll degrade requirements, the original requirements.txt works though.

Reply:
If you don't do any further training, it is compatible